azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/ComputeManager.java

```diff
@@ -59,6 +97,13 @@ public AvailabilitySets availabilitySets() {
         return availabilitySets;
     }
 
+    /**
+     * Get Azure virtual machine collection.
+     * <p/>
+     * The collection supports performing CRUD operations on Azure virtual machines
```

Comment:
I think we need to stay away from the word "collection" in describing these things, because Collection is an actual type in Java, and these are not that. The way I've been describing these accessors is simply: "Entry point to virtual machine management."